check_for_new_releases.py

```diff
@@ -0,0 +1,68 @@
+    return [o for o in os.listdir(d) 
+                    if os.path.isdir(os.path.join(d,o))]
+
+if __name__ == '__main__':
+    directories = get_subdirectories('./')
+    directories = [d for d in directories if not d.startswith('.') and not d == 'workflow']
+    
+    g = Github()
+    
+    for d in directories:
+        latest_version = get_subdirectories(d)[-1]
+        with open(os.path.join(d, latest_version, 'artifacthub-pkg.yml'), 'r') as stream:
+            try:
+                artifacthub_config = yaml.safe_load(stream)
+                repository_name = [link['url'] for link in artifacthub_config['links'] if link['name'] == 'Source'][0].replace('https://github.com/', '').partition('/tree')[0]
+                
+                repo = g.get_repo(repository_name)
+                
+                try:
+                    release = repo.get_latest_release()
+                except:
+                    print(f'{d} has no release. Skipping!')
+                    continue
+                
+                release_version = release.tag_name.strip().replace('release-', '')
+                
+                if release_version not in get_subdirectories(d):
+                    release_folder = os.path.join(d, release_version)
+                    os.makedirs(release_folder)
+                    
+                    branch_name = release.target_commitish
+
+                    # https://github.com/PyGithub/PyGithub/issues/1157#issuecomment-507498931
+                    readme = repo.get_contents(
+                        'README.md', ref=branch_name).decoded_content.decode('utf-8')
+
+                    image_regex = r'(?:\[(?P<caption>.*?)\])\((?P<path>.*?)\)'
+
+                    # Replace relative image paths with absolute paths pointing to the github repo
+                    for match in re.finditer(image_regex, readme, re.IGNORECASE | re.MULTILINE):
+                        caption, path = match.group('caption', 'path')
+                        absolute_path = urljoin(
+                            f'https://raw.githubusercontent.com/{repository_name}/{branch_name}/', path)
+                        readme = readme.replace(path, absolute_path)
+
+                    with open(os.path.join(release_folder, 'README.md'), 'w') as f:
+                        f.write(readme)
+                    
+                    datetime_format = '%Y-%m-%mT%H:%M:%SZ'
+                    
+                    artifacthub_config['version'] = release_version
+                    artifacthub_config['createdAt'] = release.created_at.strftime(datetime_format)
+                    artifacthub_config['digest'] = release.created_at.strftime(datetime_format)
+                    
+                    with open(os.path.join(release_folder, 'artifacthub-pkg.yml'), 'w') as f:
+                        yaml.dump(artifacthub_config, f, sort_keys=False)
+                    
+            except yaml.YAMLError as exc:
+                print(exc)
```

Comment:
Printing this exception here is fine, but we will never get a notification that it fails.
I propose to collect all errors and use `sys.exit(1)` or similar at the end of the script such that we also get a notification from GitHub that something failed.